### DISCUSSION TITLE
Pagination error with non-existent pages

### DISCUSSION THREAD

**Discussion by stereshko** [livewire/livewire#3085] [7. Bugs (archived)]
### Description
Quick clicks on `<button wire:click="nextPage">` take me to non-existent pages more than last page.

### Exact steps to reproduce
- Clone https://github.com/stereshko/pagination.git
- Create database `pagination`
- Edit `.env` file with your MySQL username and password
- Run `php artisan migrate:fresh --seed`
- Login into Dashboard with email `admin@admin.com` and password `qwerty`
- Click on `Livewire` tab
- Follow to the page 8 or 9
- Do several very quick (!) clicks on next page button
- You should get to page 11 or 12

**OR**

Example on Laravel Playground
[https://laravelplayground.com/#/snippets/b990af6c-cdc1-4444-b689-e86452decf71](https://laravelplayground.com/#/snippets/b990af6c-cdc1-4444-b689-e86452decf71)

**Context**
- Livewire version: 2.3.6
- Laravel version: 8.23.1
- Browser: Chrome, Firefox

**Comment by IP-Developer** [comment]
It would be easier to debug if you would set up a [Laravel Playground](https://laravelplayground.com) for this issue.

**Comment by stereshko** [comment]
> It would be easier to debug if you would set up a [Laravel Playground](https://laravelplayground.com) for this issue.

https://laravelplayground.com/#/snippets/b990af6c-cdc1-4444-b689-e86452decf71

**Comment by austenc** [comment]
@stereshko thanks for providing a github repository, very helpful!

I **cannot** reproduce the issue when I try with your repo. I'm wondering if you might have been encountering a bug and if doing a `composer update` to get the latest version of Livewire will fix it for you? Since there's no `composer.lock` in your example repo it looks like I was just testing with `2.3.8`. I was clicking very fast and tried to reproduce this for a while unsuccessfully. 

I see that you mentioned you are using Livewire `2.3.6`. _Would you mind trying with `2.3.8`?_ 

Strange thing is, I **can** reproduce this on the Laravel playground. However, I'm not sure which version of Livewire the playground currently has, so we're probably better off testing in a local repo for now.  

Let me know what you find after a `composer update`, thank you!

**Comment by stereshko** [comment]
@austenc I did `composer update` and updated Livewire in my example repo to v2.3.13. Bug still exists. Are you trying to reproduce it in the `Livewire` tab? You are wrong, my example repo have `composer.lock`.
